### PR TITLE
feat: getMeasures by surveyId

### PR DESCRIPTION
**Pull request by anaim**

### CODE REVIEW

File: src/schema/cyclic-measure/cyclic-measure.repository.spec.ts [hunk 1]
```diff
@@ -51,9 +51,9 @@ describe('CyclicMeasureRepository', () => {
 	});
 
 	test('getCyclicMeasures()', async () => {
-		const cyclicMeasures = await repo.getCyclicMeasures('__UNIT_ID__');
+		const cyclicMeasures = await repo.getCyclicMeasures('__SURVEY_ID__');
```
Comment (bennyvdhoogen) [inline src/schema/cyclic-measure/cyclic-measure.repository.spec.ts:54]: According to recently discussed conventions I would expect `findCyclicMeasures` here.

File: src/schema/cyclic-measure/cyclic-measure.repository.ts [hunk 2]
```diff
@@ -43,10 +43,10 @@ export class CyclicMeasureRepository implements ICyclicMeasureRepository {
 		return this.prisma.cyclicMeasures.create({ data });
 	}
 
-	async getCyclicMeasures(unitId: string): Promise<CyclicMeasure[]> {
+	async getCyclicMeasures(surveyId: string): Promise<CyclicMeasure[]> {
```
Comment (bennyvdhoogen) [inline src/schema/cyclic-measure/cyclic-measure.repository.ts:46]: According to recently discussed conventions I would expect `findCyclicMeasures` here.

File: src/schema/cyclic-measure/cyclic-measure.resolver.ts [hunk 3]
```diff
@@ -50,7 +50,7 @@ export class CyclicMeasureResolver {
 
 	@Query((returns) => [CyclicMeasure], { name: 'cyclicMeasures' })
 	@Roles({ roles: ['realm:aip_owner', 'realm:aip_admin'], mode: RoleMatchingMode.ANY })
-	async getSurveyCyclicMeasures(@Args('unitId', { type: () => String }) unitId: string) {
-		return this.cyclicMeasureService.getCyclicMeasures(unitId);
+	async getSurveyCyclicMeasures(@Args('surveyId', { type: () => String }) surveyId: string) {
```
Comment (bennyvdhoogen) [inline src/schema/cyclic-measure/cyclic-measure.resolver.ts:53]: According to recently discussed conventions I would expect `findSurveyCyclicMeasures` here.

File: src/schema/measure/measure.repository.spec.ts [hunk 4]
```diff
@@ -61,9 +61,9 @@ describe('MeasureRepository', () => {
 	});
 
 	test('getMeasures()', async () => {
-		const measures = await repo.getMeasures('__UNIT_ID__');
+		const measures = await repo.getMeasures('__SURVEY_ID__');
```
Comment (bennyvdhoogen) [inline src/schema/measure/measure.repository.spec.ts:64]: According to recently discussed conventions I would expect `findMeasures` here.

File: src/schema/measure/measure.repository.ts [hunk 5]
```diff
@@ -63,10 +63,10 @@ export class MeasureRepository implements IMeasureRepository {
 		return this.prisma.measures.create({ data });
 	}
 
-	async getMeasures(unitId: string): Promise<Measure[]> {
+	async getMeasures(surveyId: string): Promise<Measure[]> {
```
Comment (bennyvdhoogen) [inline src/schema/measure/measure.repository.ts:66]: According to recently discussed conventions I would expect `findMeasures` here.

File: src/schema/measure/measure.resolver.ts [hunk 6]
```diff
@@ -46,7 +46,7 @@ export class MeasureResolver {
 
 	@Query((returns) => [Measure], { name: 'measures' })
 	@Roles({ roles: ['realm:aip_owner', 'realm:aip_admin'], mode: RoleMatchingMode.ANY })
-	async getSurveyMeasures(@Args('unitId', { type: () => String }) unitId: string) {
-		return this.measureService.getMeasures(unitId);
+	async getSurveyMeasures(@Args('surveyId', { type: () => String }) surveyId: string) {
```
Comment (bennyvdhoogen) [inline src/schema/measure/measure.resolver.ts:49]: According to recently discussed conventions I would expect `findSurveyMeasures` here.

File: src/schema/cyclic-measure/cyclic-measure.repository.spec.ts (hunk 1 above)
Comment (anaim) [reply src/schema/cyclic-measure/cyclic-measure.repository.spec.ts]: Done

File: src/schema/cyclic-measure/cyclic-measure.resolver.ts (hunk 3 above)
Comment (anaim) [reply src/schema/cyclic-measure/cyclic-measure.resolver.ts:53]: Done

File: src/schema/measure/measure.repository.ts (hunk 5 above)
Comment (anaim) [reply src/schema/measure/measure.repository.ts]: Done

File: src/schema/measure/measure.repository.spec.ts (hunk 4 above)
Comment (anaim) [reply src/schema/measure/measure.repository.spec.ts]: Done

File: src/schema/measure/measure.resolver.ts (hunk 6 above)
Comment (anaim) [reply src/schema/measure/measure.resolver.ts:49]: Done

**Review by bennyvdhoogen — APPROVED** [review]

**Pull request merged**